src/Reflectivity/MetaLink.class.st

```diff
@@ -58,6 +58,7 @@ MetaLink class >> defaultOptions [
 	+ optionWeakAfter 						  "do not use #ensure: for #after "
 	- optionAnnounce                    "do we announce adding / removing links? Slow"
 	- argsAsArray                       "pass all arguments in one array. By default off as it adds an array creation"
+	+ optionStyler							  "should the metalink be visualized in the Tools?"
```

Comment:
It's quite confusing that all visible metalinks (like breakpoint) retract the optionStyler. 
What does it really mean to be visualized in Tools and what the difference from how breakpoints are show?

Reply:
yes, I am not even sure if metalinks should by default show the styler... to be improved, but the merged PR is a good first step